proto/section_response.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+import "content.proto";
+
+option java_package = "de.stroeer.model.wrapper.generated";
+option java_multiple_files = true;
+
+message SectionResponse {
+  repeated Stage stages = 1;
+}
+
+message Stage {
```

Comment:
IMO: The whole meaning of a `Stage` is now lost. Maybe we reverse the naming a bit:

```proto
message Group {
  Stage main = 1
}

message Stage {
  repeated Teaser content = 1
  StageConfig config = 2
}

message StageConfig {
 string label = 1
}
```